web/templates/message_view_header_tooltip.hbs

```diff
@@ -0,0 +1,4 @@
+<div id="message_view_header_tooltip">
```

Comment:
If we do want to separate the tooltip content from the code, we could have the tooltip template over here and reference it as a partial from `message_view_header.hbs`